### DISCUSSION TITLE
Batch updating m2m field [create new] option results in "duplicates"

### DISCUSSION THREAD

**Discussion by domdus** [directus/directus#15419] [Feature Requests]
#15034 is related and got me thinking about what [create new] on a m2m field should do and what is expected?

We have a collection articles that have a relation (m2m) to categories. Now you select 50 articles to assign to a new category "sports" by clicking [create new] in batch editing drawer.
-> Currently this results in 50 x  `sort of  duplicates`/category items created, one for each article item that went through the process.
From a editors perspective it was meant to be 1 category "sports" for the selected items put it.

To prevent this we create the category item first and then just pick it [add existing] during batch, but still it happens by mistake from time to time.

It all depends on what is the expected result.
Maybe this can be improved with batch `options` to choose the expected result of the batch:
- Create single m2m item (1 category item for 50 batch items)
- Create m2m item for each batch item (50 category items for 50 batch items) (there might be other cases where you exactly want this)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
